src/main/java/edu/illinois/cs/dt/tools/detection/detectors/ExecutingDetector.java

```diff
@@ -154,20 +154,18 @@ public void generate() {
             final double estimate = elapsed / (i + 1) * (rounds - i - 1) / 1000;
 
             if (!round.filteredTests().dts().isEmpty()) {
-                System.out.print(String.format("\r[INFO] Found %d tests in round %d of %d (%.1f seconds elapsed (%.1f total), %.1f seconds remaining).\n",
-                        round.filteredTests().size(), ++i, rounds, elapsed / 1000, totalElapsed, estimate));
+                System.out.println(
+                        buildResultString(round.filteredTests().size(), ++i, rounds,
+                                          elapsed / 1000, totalElapsed, estimate));
                 result.addAll(round.filteredTests().dts());
-		if (!roundsAreTotal){
-		    i = 0;
-		}
+                if (!roundsAreTotal){
```

Comment:
Add a space between the ")" and the "{" (I see it was like that before, but we can fix it now in this PR).

Reply:
Done